docs/tutorials/hello-minikube/hello-minikube.md

```diff
@@ -0,0 +1,265 @@
+---
+---
+
+{% capture overview %}
+
+The goal of this codelab is for you to turn a simple Hello World node.js app 
+into a replicated application running on Kubernetes. We will show you how to 
+take code that you have developed on your machine, turn it into a Docker
+container image, and then run that image on [Minikube](/docs/getting-started-guides/minikube).
+
+Kubernetes is an open source project which can run on many different environments, 
+from laptops to high-availability multi-node clusters, from public clouds to 
+on-premise deployments, from virtual machines to bare metal. Minikube provides
+a simple way of running Kubernetes on your local workstation for free.
+
+{% endcapture %}
+
+{% capture objectives %}
+
+* Run a hello world Node application in Docker
+* Deploy the application to Minikube.
+* View application logs
+* Update the application image
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture lessoncontent %}
+
+## Create your Node.js application
+
+The first step is to write the application. Save this code in a folder called "`hellonode/`" with the filename `server.js`:
+
+#### server.js
+
+```javascript
+var http = require('http');
+var handleRequest = function(request, response) {
+  console.log('Received request for URL: ' + request.url);
+  response.writeHead(200);
+  response.end('Hello World!');
+};
+var www = http.createServer(handleRequest);
+www.listen(8080);
+```
```

Comment:
I like that we're just telling people to copy this into a file. Making the tutorial seem as light as possible. However, This should still be checked into a server.js file on the site, and use a include tag in the doc